.github/workflows/ci-linux.yml

```diff
@@ -32,18 +32,11 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v4
 
-    - name: Setup vcpkg caching
-      uses: actions/github-script@v7
+    - name: Setup vcpkg
+      uses: ./.github/actions/setup-vcpkg
```

Comment:
btw, might be good to move this to the actions repo, so we can fix bugs and issues (that are non-breaking) in a central place, updating all forks etc. Just like we do for a lot of the other stuff :)
Doesn't have to be in this PR, etc etc.